database/migration/src/main/resources/metadata_changesets/partition_metadata_table.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog objectQuotingStrategy="QUOTE_ALL_OBJECTS"
+                   xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
+
+    <changeSet id="update_indexes_metadata_table" author="cahrens" dbms="mysql,postgresql,hsqldb,mariadb">
+        <comment>Updates indexes on the METADATA_ENTRY table</comment>
```

Comment:
Could we be more descriptive here? Eg "indexes on metadata keys within a workflow and within a specific call instances"

Reply:
I updated the comment to state that the indexes are related to the primary key change.